docs/book/src/development/releasing.md

```diff
@@ -7,14 +7,18 @@
 
 ## Create tag, and build staging container images
 
-1. Create a new local repository of <https://github.com/kubernetes-sigs/cluster-api-provider-aws> (e.g. using `git clone`).
+1. Create a new local repository of <https://github.com/kubernetes-sigs/cluster-api-provider-aws> (e.g. using `git clone`). __**Note**__: This needs a be a fork.
```

Comment:
Unclear. If you meant `needs to be`, then we should write

```
Please fork <https://github.com/kubernetes-sigs/cluster-api-provider-aws> and clone your own repository with e.g. `git clone git@github.com:YourGitHubUsername/cluster-api-provider-aws.git`.
```

And in any case, please state _why_ this is a requirement since otherwise this detail can never be removed again, or fully understood by others.